lib/infrastructure/profile/profile_repository.dart

```diff
@@ -159,4 +159,42 @@ class ProfileRepository implements IProfileRepository {
 
     return null;
   }
+
+  // update user name
+  @override
+  Future<Either<ProfileFailure, Unit>> updateUsername(
+      {String? firstname, String? lastname}) async {
```

Comment:
Do you not get static analyzer errors in your IDE? It should complain that you're missing a trailing comma here after `lastname,`

Reply:
fixed all analyzer issues raised by current changes,